.github/workflows/build.yml

```diff
@@ -78,9 +73,9 @@ jobs:
           MATRIX=$(
             {
               cibuildwheel --print-build-identifiers --platform linux \
-                | sed 's/.*/{"cibw-only": "&", "os": "ubuntu-20.04"}/' \
+                | jq --raw-input --compact-output --null-input '{"cibw-only": inputs, "os": "ubuntu-20.04"}' \
```

Comment:
Optional, I felt this is more consistent, but understandable if the sed seems better.

Reply:
Yeah, I think I'd be inclined to stick with sed here -- although jq usually makes working with JSON nicer, the sed is slightly easier for me to parse.